keyboards/EU_ISOlation/config.h

```diff
@@ -0,0 +1,33 @@
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```

Comment:
```suggestion
#define PRODUCT_ID      0x4373
```

From hash generated:
```sh
echo -n "EU ISOlation" | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```